Fix/return path (#1)
imobanco/django-ses: README.rst

  http://readthedocs.org/docs/boto/en/latest/ref/ses.html#boto.ses.regions
  http://docs.aws.amazon.com/general/latest/gr/rande.html

``AWS_SES_FROM_EMAIL``

>   Optional. The email addres use as the "From" address for the email. The address that you specify has to be verified.  

Reviewer: ```suggestion
  Optional. The email addres to be used as the "From" address for the email. The address that you specify has to be verified.  
```

  http://readthedocs.org/docs/boto/en/latest/ref/ses.html#boto.ses.regions
  http://docs.aws.amazon.com/general/latest/gr/rande.html

``AWS_SES_FROM_EMAIL``

>   Optional. The email addres use as the "From" address for the email. The address that you specify has to be verified.  

Reviewer 2: ```suggestion
  Optional. Use this to set the sender e-mail address.
  The address that you specify has to be verified.
```

  For more information please refer to http://aws.amazon.com/ses/faqs/#38

``AWS_SES_RETURN_PATH``

>   Optional. Use `AWS_SES_RETURN_PATH` to receive complaint notifications
>   It only works in the v2 client.

Reviewer 2: Padronizar com as outras documentações:
```suggestion
  Optional. Use this to set an e-mail address to receive complaint notifications.
  It only works in the v2 client.
```

``AWS_SES_RETURN_PATH``
  Instruct Amazon SES to forward bounced emails and complaints to this email.

>   Works only v2 client.

Reviewer 2: Eu escreveria de uma forma mais formal.
```suggestion
  It only works in the v2 client.
```
